locale/ar/site.json

```diff
@@ -165,7 +165,7 @@
     "index": {
```

Comment:
Nothing but just a trick.

Yon don't need to write 'startDate', 'endDate' and 'link', because it will be automatically in sync with the English version, what you can do is just to keep your translations. Otherwises you have to update datatimes, as well as links.

 :)

Reply:
Thanks @MaledongGit